types/options.go

```diff
@@ -48,6 +48,12 @@ func loadDefaultStoreOptions() {
 			loadDefaultStoreOptionsErr = err
 			return
 		}
+	} else if path, ok := os.LookupEnv("XDG_CONFIG_HOME"); ok {
+		defaultOverrideConfigFile = filepath.Join(path, "containers/storage.conf")
```

Comment:
Pass the "containers" and "storage.conf" path elements to `Join()` as two arguments, not one.

Reply:
Done